talisker/__init__.py

```diff
@@ -42,6 +43,11 @@
 
 def initialise(env=os.environ):
     config = get_config(env)
+    # set this early so any imports of prometheus client will be imported
+    # correctly
+    if 'prometheus_multiproc_dir' not in os.environ:
```

Comment:
Maybe this should be preceded by a check `if pkg_is_installed('prometheus-client')`?

Reply:
Maybe, but there's no harm in setting the environment variable if prometheus-client isn't installed, it will do nothing.

Reply:
It's not just setting the envvar but creating a temp dir unnecessarily.

Reply:
Sure, but I don't have an issue with that particularly, but have added the protection.